app/services/idv/step.rb

```diff
@@ -3,7 +3,7 @@ module Idv
   class Step
     include VendorValidated
 
-    def initialize(analytics:, idv_form:, idv_session:, params:)
+    def initialize(analytics: nil, idv_form:, idv_session:, params:)
```

Comment:
Moving analytics back into the controller is a fair change, and consistent with how we do things elsewhere.